### PR TITLE
Remove/deprecate wrap method

### LINKED ISSUE #268: Remove/deprecate wrap method
Everything that we need can be implemented by HOC, but HOC can't be implemented by `wrap`.

### PR DESCRIPTION
**Pull request by yarastqt**
Resolve #268

### CODE REVIEW

**Comment by yarastqt** [comment]
Also need remove from docs

File: packages/react/src/index.ts [hunk 1]
```diff
@@ -265,10 +265,13 @@ export abstract class Block<P = {}, S = {}> extends Component<EntityProps<P>, S>
...
-        const { props, state } = this;
-        const optionalyReplaced = this.replace(props, state);
+        const tag = this.tag(this.props, this.state);
+        const children = this.content(this.props, this.state);
+        const attrs = this.attrs(this.props, this.state);
+        const style = this.style(this.props, this.state);
+        const extendedAttributes = { ...attrs, style: { ...attrs.style, ...style } };
 
-        return this.wrap(props, state, optionalyReplaced);
+        return this.prerender(tag, extendedAttributes, children);
```
Comment (awinogradov) [inline packages/react/src/index.ts]: It's useful method without `wrap` and `replace`. Let's Remove it too ;)

Comment (yarastqt) [reply packages/react/src/index.ts]: Nope, because prerender get context from `Block` and `Elem`

Comment (awinogradov) [reply packages/react/src/index.ts]: I know my friend) Let's move logic here ;)

**Pull request merged**